### PR TITLE
Reduce crud in travis-ci log

### PR DESCRIPTION
**Pull request by bdemers**

### CODE REVIEW

File: src/ci/build.sh [hunk 1]
```diff
@@ -21,7 +21,7 @@ COMMON_SCRIPT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/common.sh"
 source "${COMMON_SCRIPT}"
 
 # all the prep is done, lets run the build!
-MVN_CMD="mvn -s src/ci/settings.xml"
+MVN_CMD="mvn -s src/ci/settings.xml -B -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn"
```
Comment (mraible) [inline src/ci/build.sh]: You can also use `mvn -q` to "quiet" Maven.

Comment (bdemers) [reply src/ci/build.sh]: Yeah, I want some of the output so i can debug an odd build, i just wish there was a `--verbose 2` or something flag

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+indent_size=2
+
+[.eslintrc]
+indent_style=space
+indent_size=2
+
+[tslint.json]
+indent_style=space
+indent_size=2
+
```
Comment (jmelberg-okta) [inline .editorconfig]: nit: You can remove the extra newline here

File: .editorconfig [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+[*.coffee]
+indent_style=space
+indent_size=2
+
+[*.less]
+indent_style=space
+indent_size=2
+
+[*.sass]
```
Comment (jmelberg-okta) [inline .editorconfig]: nit: I don't think you need to worry about `.sass` files.

**Comment by bdemers** [comment]
Yeah, i generated that .editorconfig, looks like a lot of that stuff is duplicated
i'll clean it up

**Comment by bdemers** [comment]
@jmelberg-okta one more time :)

**Pull request merged**